### PR TITLE
Temporary Files Cleaner

### PR DESCRIPTION
**Pull request by tnkrishnank**

### CODE REVIEW

File: temporary_files_cleaner/temporary_files_cleaner.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sudo apt-get autoremove -y
+sudo apt-get autoclean
+sudo apt-get clean
+sudo journalctl --vacuum-time=3d
```
Comment (suraj-singh12) [inline temporary_files_cleaner/temporary_files_cleaner.sh:6]: 1. What does this command do in the background?
2. What type of files and from where does it delete?
3. Does it deal with user-level files or `system` files?
4. What does the switch `--vacuum-time=3d` do and why the value passed is `3d` only?

File: temporary_files_cleaner/temporary_files_cleaner.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+sudo rm -rf ~/.cache/thumbnails/*
```
Comment (suraj-singh12) [inline temporary_files_cleaner/temporary_files_cleaner.sh:7]: 1. What does this do?
2. Does erasing the files here impact the performance of the software/tool using this location?

File: temporary_files_cleaner/temporary_files_cleaner.sh (hunk 1 above)
Comment (tnkrishnank) [reply temporary_files_cleaner/temporary_files_cleaner.sh:6]: Basically those are system log files.
There are two types in which these logs can be deleted. One is by time and other by size. Here, 3d means 3 days. That is only the logs of the past three days are retained and the rest are removed.

File: temporary_files_cleaner/temporary_files_cleaner.sh (hunk 2 above)
Comment (tnkrishnank) [reply temporary_files_cleaner/temporary_files_cleaner.sh:7]: It is used to clear the cache.
Yes for sure, it reduces the performance of the software/tool using the location. It is the basic concept of cache. But the performance reduction is minute and mostly cannot be noticed by the user.

Comment (suraj-singh12) [reply temporary_files_cleaner/temporary_files_cleaner.sh:7]: Good then!

File: temporary_files_cleaner/temporary_files_cleaner.sh (hunk 1 above)
Comment (tnkrishnank) [reply temporary_files_cleaner/temporary_files_cleaner.sh:6]: I accidentally deleted the repository.

**Pull request closed without merging**